### PR TITLE
move supported tokens config to a json file

### PR DESCRIPTION
**Pull request by BruceLiux**
### Motivation
Config supported token in code is not ideal since (1) it will leak internal information (2) takes time for PR communication  when we need to update token list. 

### Solution
Move token config to a file so that we could override that file in coinbase rosetta codebase. As a result, we could avoid info leak and speed up the process when adding new tokens. Also add a env flag so that we could bypass the entire whitelist with configuration.

### CODE REVIEW

File: optimism/client.go [hunk 1]
```diff
@@ -565,7 +568,8 @@ func (ec *Client) erc20TokenOps(
 			continue
 		}
 
-		if _, ok := ec.supportedTokens[strings.ToLower(contractAddress)]; !ok {
+		_, ok = ec.supportedTokens[strings.ToLower(contractAddress)]
+		if ec.filterTokens && !ok {
```
Comment (Inphi) [inline optimism/client.go:572]: This changes the default behavior of whitelisting erc20 tokens. Just wanna point this point to make sure this is this desired.

**Review by Inphi — COMMENTED** [review]
Sorry for the late review. Thanks for doing this.

File: tokenList.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "0x94b008aa00579c1307b0ef2c499ad98a8ce58e58": true,
+    "0x68f180fcce6836688e9084f035309e29bf0a2095": true,
+    "0x7f5c764cbc14f9669b88837ca1490cca17c31607": true
+  },
+  "Testnet" : {
+    "0x4200000000000000000000000000000000000042": true,
+    "0xda10009cbd5d07dd0cecc66161fc93d7c9000da1": true,
+    "0x94b008aa00579c1307b0ef2c499ad98a8ce58e58": true,
+    "0x68f180fcce6836688e9084f035309e29bf0a2095": true,
+    "0x7f5c764cbc14f9669b88837ca1490cca17c31607": true
```
[the remark this answers is not in the document]
Comment (BruceLiux) [reply tokenList.json]: Great catch! Updated testnet token address to the correct one.

File: optimism/client.go (hunk 1 above)
Comment (BruceLiux) [reply optimism/client.go:572]: yeah this is desired. We want to bypass the whilelist in rosetta.

**Review by Inphi — APPROVED** [review]
Thank you

**Pull request merged**